src/Raven.Server/Documents/Replication/Incoming/IncomingReplicationHandler.cs

```diff
@@ -317,7 +315,42 @@ public MergedDocumentReplicationCommand(DataForReplicationCommand replicationInf
                 _isInternal = isInternal;
             }
 
-            protected virtual ChangeVector PreProcessItem(DocumentsOperationContext context, ReplicationBatchItem item)
+            protected ChangeVector PreProcessItem(DocumentsOperationContext context, ReplicationBatchItem item)
+            {
+                PreProcessAttachments(context, item);
+
+                return PreProcessItemInternal(context, item);
```

Comment:
why do you needed to rename it?
simply `base.PreProcessItem(..)` ?

Reply:
done